helpers/typescript.ts

```diff
@@ -115,9 +115,37 @@ export const installTSTemplate = async ({
     console.log(
       `\nUsing managed index from LlamaCloud. Ensure the ${yellow("LLAMA_CLOUD_* environment variables are set correctly.")}`,
     );
+    // copy llamacloud pipeline code for documents
+    await copy(
```

Comment:
alternatively, we could not generate LlamaCloud specific code and check at runtime if we're having LlamaCloud configured (and then run the pipeline).
@leehuwuj what are your thoughts?
please make sure that it's the same behavior in the python code

Reply:
in Python i also run the different pipeline based on the index type, so i'd put the code in the same file. We then no need to add this copy logic.